Makefile

```diff
@@ -27,6 +27,7 @@ bin/rtf: tmp_rtf_bin.tar | bin
 	rm $<
 	touch $@
 
+.DELETE_ON_ERROR:
```

Comment:
This placement suggests it applies only to `tmp_rtf_bin.tar` when actually this has global effect. I suggest putting it right at the top before the `.PHONY: default all` and with a blank line between `.DELETE_ON_ERROR` and `.PHONY`.

Reply:
Thanks. Done!